lib/evss/claims_service.rb

```diff
@@ -3,8 +3,8 @@
 
 module EVSS
   class ClaimsService < BaseService
-    def claims
-      get 'vbaClaimStatusService/getOpenClaims'
+    def all_claims
+      get 'vbaClaimStatusService/getClaims'
```

Comment:
Did you see any difference between getClaims and getOpenClaims apart from the latter having less information?